packages/project-access/src/project/info.ts

```diff
@@ -203,3 +203,14 @@ export async function getProjectType(projectRoot: string): Promise<ProjectType>
     }
     return capType;
 }
+
+/**
+ * Reads the package.json file and returns its content.
+ *
+ * @param {string} packageJsonPath - The path to the package.json file.
+ * @param {Editor} fs - The file system editor.
+ * @returns {Package} The content of the package.json file.
+ */
+export function getPackageJson(packageJsonPath: string, fs?: Editor): Package {
+    return fs?.readJSON(packageJsonPath) as Package;
+}
```

Comment:
we should also handle the case if `fs?: Editor` is not provided, the low level file access functions already support this. We could do:

```suggestion
export async function getPackageJson(packageJsonPath: string, fs?: Editor): Promise<Package> {
    return readJSON<Package>(packageJsonPath, fs);
}
```

Note that the function needs to be async then.

Reply:
I have updated this functionality with async. thanks.